src/commands/synthetics/__tests__/run-tests-lib.test.ts

```diff
@@ -190,7 +191,7 @@ describe('run-test', () => {
       await expect(
         runTests.executeTests(mockReporter, {
           ...ciConfig,
-          global: configOverride,
+          defaultTestOverrides: configOverride,
```

Comment:
I made https://github.com/DataDog/datadog-ci/pull/1340 to handle changes like this